src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\FunctionNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException;
+use PhpCsFixer\Fixer\ConfigurableFixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ */
+final class NativeFunctionInvocationFixer extends AbstractFixer implements ConfigurableFixerInterface
+{
+    /**
+     * @var string[]
+     */
+    private $exclude = array();
+
+    /**
+     * @var array
+     */
+    private static $defaultConfiguration = array(
+        'exclude' => array(),
+    );
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            $this->exclude = array();
+
+            return;
+        }
+
+        $key = 'exclude';
+
+        if (!\array_key_exists($key, $configuration) || !\is_array($configuration[$key])) {
+            throw new InvalidFixerConfigurationException(
+                $this->getName(),
+                \sprintf(
+                    'Configuration must define "%s" as an array.',
+                    $key
+                )
+            );
+        }
+
+        foreach ($configuration[$key] as $functionName) {
+            if (!\is_string($functionName) || \trim($functionName) === '' || \trim($functionName) !== $functionName) {
+                throw new InvalidFixerConfigurationException(
+                    $this->getName(),
+                    \sprintf(
+                        'Each element must be a non-empty, trimmed string, got "%s" instead.',
+                        \is_object($functionName) ? \get_class($functionName) : \gettype($functionName)
+                    )
+                );
+            }
+        }
+
+        $this->exclude = $configuration[$key];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, Tokens $tokens)
+    {
+        static $internalFunctionNames = null;
+
+        if (null === $internalFunctionNames) {
+            $internalFunctionNames = $this->getInternalFunctionNames();
```

Comment:
what if I want to add a `\` to a function I declared in the global namespace but didn't load with the Fixer project? (i.e. non-native function but still declared in global namespace?)